### PR TITLE
 Create firefly.json

### PR DESCRIPTION
**Pull request by bestZwei**
For study and test, I am a Uni student... I wont abuse it

### CODE REVIEW

File: domains/firefly.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Vibred",
+        "email": "vibred@pm.me"
+    },
+    "record": {
+        "NS": ["cora.ns.cloudflare.com", "jonah.ns.cloudflare.com"]
```
Comment (CuteDog5695) [inline domains/firefly.json:7]: CI doesnt support NS records at the moment

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/firefly.json (hunk 1 above)
Comment (Vibred) [reply domains/firefly.json:7]: > CI heyhey... I am in the waiting list! 我只是在排队

Comment (CuteDog5695) [reply domains/firefly.json:7]: NS will probably be used for maintainers and trusted users, but im not sure yet

Comment (CuteDog5695) [reply domains/firefly.json:7]: > heyhey... I just join in the waiting list ! See if Andrew let us use NS record

This is is-a.dev not Open Domains mate

Comment (Vibred) [reply domains/firefly.json:7]: I know,,by the side, can you help me check the request code here?
https://github.com/open-domains/register/pull/1164/commits
O,O

**Pull request closed without merging**